validation/standard/versions.go

```diff
@@ -7,13 +7,19 @@ import (
 type Tag string
 
 type (
-	VersionTags            = map[Tag]superchain.ContractVersions
 	BytecodeHashTags       = map[Tag]L1ContractBytecodeHashes
 	BytecodeImmutablesTags = map[Tag]ContractBytecodeImmutables
 )
 
+type VersionTags struct {
+	Tags    map[Tag]superchain.ContractVersions `toml:"contract_tag"`
+	Version string                              `toml:"op_contracts_release,omitempty"`
```

Comment:
Nit: 
```suggestion
	Version Tag                              `toml:"op_contracts_release,omitempty"`
```

Reply:
Could we call this `StandardVersion` or something?

In general it is a bit confusing to have a single enshrined standard version tag, but then to display detailed (per contract) version information for multiple tags. Are we planning on using the "other" tag information for anything?

Reply:
Renamed per above